input/resources/au-core-medicationrequest.xml

```diff
@@ -247,8 +247,8 @@
       <constraint>
         <key value="au-core-medreq-03"/>
         <severity value="error"/>
-        <human value="Date is precise to the day or, if not available, the Data Absent Reason extension shall be present"/>
-        <expression value="(toString().length() &gt;= 10) xor extension('http://hl7.org/fhir/StructureDefinition/data-absent-reason').exists()"/>
+        <human value="Date shall be precise to the day or, if not available, the Data Absent Reason extension shall be present"/>
+        <expression value="(toString().length() &gt;= 8) xor extension('http://hl7.org/fhir/StructureDefinition/data-absent-reason').exists()"/>
```

Comment:
Incorrect inclusion - this is only supposed to be a change to precision length, NOT to xor data absent reason.